tests-scan

```diff
@@ -200,7 +200,7 @@ def queue_test(priority, name, number, revision, ref, context, base,
         logging.info("Published '{0}' on '{1}' with command: '{2}'".format(name, revision, command))
 
 
-def prioritize(status, title, labels, priority, context, number):
+def prioritize(status, title, labels, priority, context, number, direct):
```

Comment:
So I guess the by now ancient polling method for `bots/tests-scan` would also be considered direct.